gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -300,31 +357,17 @@ public GrpcCallContext withRequestParamsDynamicHeaderOption(String requestParams
     return withCallOptions(newCallOptions);
   }
 
-  @InternalApi("for testing")
-  Deadline getDeadline() {
-    return callOptions.getDeadline();
-  }
-
-  @Override
-  public boolean equals(Object o) {
```

Comment:
Why did you remove equals/hashcode?

Reply:
This comes from discussions earlier, I tagged you in the doc.

Reply:
Ok